Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import Deferred
 
 struct PlaceUtilities {
```

Comment:
nit: I assume `PlaceUtilities.sort(...:byDistanceFromLocation` is unused now – shall we remove it?